src/ussd_higherhealth_healthcheck.js

```diff
@@ -224,6 +225,714 @@ go.app = (function () {
+                "Kingsway Auckland Park (APK)",
+                "Kingsway Bunting Road (APB)",
+                "Soweto"
+        ],
+            "University of Kwazulu Natal (UKZN)":[
+                "Edgewood",
+                "Howard",
+                "Medical School",
+                "Pietermaritzburg",
+                "Westville"
+        ],
+            "University of Limpopo (UL)":[
+                "Turfloop Campus"
+        ],
+            "University of Mpumalanga (UMP)":[
+                "Mbombela",
+                "Siyabuswa"
+
+        ],
+            "University of Pretoria (UP)":[
+                "GIBS",
+                "Groenkloof",
+                "Hatfield (Main Campus)",
+                "Hillcrest",
+                "Mamelodi",
+                "Onderstepoort",
+                "Prinshof"
+
+        ],
+            "University of Venda (UNIVEN)":[
+                "Thohoyandou"
+
+        ],
+            "University of Western Cape (UWC)":[
+                "Bellville"
+
+        ],
+            "University of Zululand (UNIZULU)":[
+                "KwaDlangezwa",
+                "Richards Bay"
+
+        ],
+            "University of the Free State (UFS)":[
+                "Main Campus",
+                "Qwaqwa Campus",
+                "South Campus"
+
+        ],
+            "University of the Witwatersrand (WITS)":[
+                "Braamfontein"
+
+        ],
+            "Vaal University of Technology (VUT)":[
+                "Ekurhuleni",
+                "Main Campus (Vanderbijlpark)",
+                "Secunda"
+
+        ],
+            "Vhembe":[
+                "Central Office",
+                "Makwarela",
+                "Mashamba",
+                "Mavhoi",
+                "Musina",
+                "Shingwedzi",
+                "Thengwe",
+                "Tshisimani"
+
+        ],
+            "Vuselela":[
+                "Central Office",
+                "Jouberton",
+                "Klerksdorp",
+                "Matlosana",
+                "Potchefstroom",
+                "Taung"
+
+        ],
+            "Walter Sisulu University (WSU)":[
+                "Buffalo City",
+                "Butterworth (Ibika)",
+                "Mthatha",
+                "Queenstown (Masibulele)"
+
+        ],
+            "Waterberg":[
+                "Business",
+                "Central Office",
+                "Engineering",
+                "IT&C",
+                "Thabazimbi"
+
+        ],
+            "West Coast":[
+                "Atlantis",
+                "Central Office",
+                "Citrusdal",
+                "Malmesbury",
+                "Vredenburg",
+                "Vredendal"
+
+        ],
+            "Western":[
+                "Carletonville",
+                "Central Office",
+                "Krugersdorp",
+                "Krugersdorp West",
+                "Randfontein"
+
+        ],
+            "Other":[
+                "Other"
+            ],
```

Comment:
We have "Other" here, but what if the user selects a University, but wants to select an "Other" for campus, because their campus isn't on the list?

I think we should also have "Other" in all the lists of campuses. Probably added to the `choices` list, instead of manually adding it to each of these lists.

We probably also want some testing for the "Other" logic.